### PR TITLE
[skip-ci] Add cftime groupby, resample benchmarks

### PR DESCRIPTION
**Pull request by dcherian**
xref #7730

### CODE REVIEW

File: asv_bench/benchmarks/groupby.py [hunk 1]
```diff
@@ -132,3 +132,34 @@ def setup(self, *args, **kwargs):
         super().setup(**kwargs)
         self.ds1d = self.ds1d.chunk({"time": 50})
         self.ds2d = self.ds2d.chunk({"time": 50, "z": 4})
+
+
+class ResampleCFTime:
```
Comment (headtr1ck) [inline asv_bench/benchmarks/groupby.py:137]: I think you are missing some class inheritance or otherwise there won't be any benchmark?

**Comment by dcherian** [comment]
I'm not sure why asv can't create  the env for benchmarking.


```
+ echo 'Baseline:  25d9a28e12141b9b5e4a79454eb76ddd2ee2bc4d (pydata:main)'
ram [7110632]: Baseline:  25d9a28e12141b9b5e4a79454eb76ddd2ee2bc4d (pydata:main)
+ echo 'Contender: 4ca69efdf7e5e3fc661e5ec3ae618d102a374f32 (dcherian:bench-cftime-groupby)'
Contender: 4ca69efdf7e5e3fc661e5ec3ae618d102a374f32 (dcherian:bench-cftime-groupby)
++ which conda
+ export CONDA_EXE=/usr/bin/conda
+ CONDA_EXE=/usr/bin/conda
+ ASV_OPTIONS='--split --show-stderr --factor 1.5'
+ asv continuous --split --show-stderr --factor 1.5 25d9a28e12141b9b5e4a79454eb76ddd2ee2bc4d 4ca69efdf7e5e3fc661e5ec3ae618d102a374f32
+ tee benchmarks.log
+ sed '/Traceback \|failed$\|PERFORMANCE DECREASED/ s/^/::error::/'
Traceback (most recent call last):
  File "/home/runner/micromamba-root/envs/xarray-tests/bin/asv", line 10, in <module>
    sys.exit(main())
  File "/home/runner/micromamba-root/envs/xarray-tests/lib/python3.10/site-packages/asv/main.py", line 38, in main
    result = args.func(args)
  File "/home/runner/micromamba-root/envs/xarray-tests/lib/python3.10/site-packages/asv/commands/__init__.py", line 49, in run_from_args
    return cls.run_from_conf_args(conf, args)
  File "/home/runner/micromamba-root/envs/xarray-tests/lib/python3.10/site-packages/asv/commands/continuous.py", line 75, in run_from_conf_args
    return cls.run(
  File "/home/runner/micromamba-root/envs/xarray-tests/lib/python3.10/site-packages/asv/commands/continuous.py", line 114, in run
    result = Run.run(
  File "/home/runner/micromamba-root/envs/xarray-tests/lib/python3.10/site-packages/asv/commands/run.py", line 294, in run
    Setup.perform_setup(environments, parallel=parallel)
  File "/home/runner/micromamba-root/envs/xarray-tests/lib/python3.10/site-packages/asv/commands/setup.py", line 89, in perform_setup
    list(map(_create, environments))
  File "/home/runner/micromamba-root/envs/xarray-tests/lib/python3.10/site-packages/asv/commands/setup.py", line 21, in _create
    env.create()
  File "/home/runner/micromamba-root/envs/xarray-tests/lib/python3.10/site-packages/asv/environment.py", line 704, in create
    self._setup()
  File "/home/runner/micromamba-root/envs/xarray-tests/lib/python3.10/site-packages/asv/plugins/conda.py", line 174, in _setup
    self._run_conda(['env', 'create', '-f', env_file_name,
  File "/home/runner/micromamba-root/envs/xarray-tests/lib/python3.10/site-packages/asv/plugins/conda.py", line 227, in _run_conda
    return util.check_output([conda] + args, env=env)
  File "/home/runner/micromamba-root/envs/xarray-tests/lib/python3.10/site-packages/asv/util.py", line 754, in check_output
    raise ProcessError(args, retcode, stdout, stderr)
asv.util.ProcessError: Command '/usr/bin/conda env create -f /tmp/tmphnyugp42.yml -p /home/runner/work/xarray/xarray/asv_bench/.asv/env/df282ba4a530a0853b7f9108ec3ff02d --force' returned non-zero exit status 1
· Creating environments
·· Error running /usr/bin/conda env create -f /tmp/tmphnyugp42.yml -p /home/runner/work/xarray/xarray/asv_bench/.asv/env/df282ba4a530a0853b7f9108ec3ff02d --force (exit status 1)
   STDOUT -------->
   
   STDERR -------->
   
   SpecNotFound: /tmp/tmphnyugp42.yml is not a valid yaml file.
```

**Comment by Illviljan** [comment]
Feels like it worked quite recently, has something changed? Benchmarks still works on the flox side.

**Comment by dcherian** [comment]
Yes very frustrating it broke between [5](https://github.com/pydata/xarray/actions/runs/4810343177/jobs/8562834120) and [3](https://github.com/pydata/xarray/actions/runs/4831637514/jobs/8609338116) days ago

**Comment by dcherian** [comment]
Here's  a truncated diff:
```
5c5
<  Job is about to start running on the hosted runner: GitHub Actions 2 (hosted)
---
>  Job is about to start running on the hosted runner: GitHub Actions 4 (hosted)
14,16c14,16
<  Version: 20230417.1
<  Included Software: https://github.com/actions/runner-images/blob/ubuntu20/20230417.1/images/linux/Ubuntu2004-Readme.md
<  Image Release: https://github.com/actions/runner-images/releases/tag/ubuntu20%2F20230417.1
---
>  Version: 20230426.1
>  Included Software: https://github.com/actions/runner-images/blob/ubuntu20/20230426.1/images/linux/Ubuntu2004-Readme.md
>  Image Release: https://github.com/actions/runner-images/releases/tag/ubuntu20%2F20230426.1
64c64
<  git version 2.40.0
---
>  git version 2.40.1
66c66
<  Temporarily overriding HOME='/home/runner/work/_temp/f08ac219-4ee6-4fb1-962a-95a90ed2ab7a' before making global git config changes
---
>  Temporarily overriding HOME='/home/runner/work/_temp/ea752478-2e4c-4c11-9a80-64b3b9b03d35' before making global git config changes
575c593
<  0f4e99d036b0d6d76a3271e6191eacbc9922662f
---
>  a220022e7ef8d3df68619643954352fa39394ea8
585c603
<  '0f4e99d036b0d6d76a3271e6191eacbc9922662f'
---
>  'a220022e7ef8d3df68619643954352fa39394ea8'
605,607c623,625
<  Received 5491626 of 5491626 (100.0%), 32.9 MBs/sec
<  Cache Size: ~5 MB (5491626 B)
<  [command]/usr/bin/tar -xf /home/runner/work/_temp/d8162eae-90d3-4277-abdf-d3574b623c16/cache.tgz -P -C /home/runner/work/xarray/xarray -z
---
>  Received 5491632 of 5491632 (100.0%), 8.7 MBs/sec
>  Cache Size: ~5 MB (5491632 B)
>  [command]/usr/bin/tar -xf /home/runner/work/_temp/8bf4827f-8427-494a-86a7-047343a5e85a/cache.tgz -P -C /home/runner/work/xarray/xarray -z
609c627
<  Cache hit for key 'micromamba-bin https://micro.mamba.pm/api/micromamba/linux-64/latest Wed Apr 26 2023 YYY'
---
>  Cache hit for key 'micromamba-bin https://micro.mamba.pm/api/micromamba/linux-64/latest Fri Apr 28 2023 YYY'
634c652
<  Modifying RC file "/tmp/micromamba-m6EeL6/.bashrc"
---
>  Modifying RC file "/tmp/micromamba-zQHzJD/.bashrc"
637c655
<  Adding (or replacing) the following in your "/tmp/micromamba-m6EeL6/.bashrc" file
---
>  Adding (or replacing) the following in your "/tmp/micromamba-zQHzJD/.bashrc" file
772c790
<    + coverage                          7.2.3  py310h1fa729e_0          conda-forge/linux-64     282kB
---
>    + coverage                          7.2.4  py310h2372a71_0          conda-forge/linux-64     280kB
813c831
<    + hypothesis                       6.74.0  pyha770c72_0             conda-forge/noarch       291kB
---
>    + hypothesis                       6.74.1  pyha770c72_0             conda-forge/noarch       292kB
920c938
<    + platformdirs                      3.3.0  pyhd8ed1ab_0             conda-forge/noarch        18kB
---
>    + platformdirs                      3.5.0  pyhd8ed1ab_0             conda-forge/noarch        19kB
954c972
<    + requests                         2.28.2  pyhd8ed1ab_1             conda-forge/noarch        57kB
---
>    + requests                         2.29.0  pyhd8ed1ab_0             conda-forge/noarch        57kB
985c1003
<    + virtualenv                      20.22.0  pyhd8ed1ab_0             conda-forge/noarch         3MB
---
>    + virtualenv                      20.23.0  pyhd8ed1ab_0             conda-forge/noarch         3MB
1029,1030d1046
<  Linking font-ttf-inconsolata-3.000-h77eed37_0
<  Linking font-ttf-source-code-pro-2.038-h77eed37_0
1032a1049,1050
>  Linking font-ttf-inconsolata-3.000-h77eed37_0
>  Linking font-ttf-source-code-pro-2.038-h77eed37_0
1053d1070
<  Linking xorg-xextproto-7.3.0-h0b41bf4_1003
1054a1072,1073
>  Linking xorg-xextproto-7.3.0-h0b41bf4_1003
>  Linking xxhash-0.8.1-h0b41bf4_0
1056a1076
>  Linking libaec-1.0.6-hcb278e6_1
1057a1078,1079
>  Linking libopenblas-0.3.21-pthreads_h78a6416_3
>  Linking lz4-c-1.9.4-hcb278e6_0
1066a1089,1091
>  Linking c-ares-1.18.1-h7f98852_0
>  Linking keyutils-1.6.1-h166bdaf_0
>  Linking openssl-3.1.0-hd590300_2
1069,1070d1093
<  Linking ncurses-6.3-h27087fc_1
<  Linking lz4-c-1.9.4-hcb278e6_0
1072d1094
<  Linking xxhash-0.8.1-h0b41bf4_0
1078,1079c1100
<  Linking icu-70.1-h27087fc_0
<  Linking libopenblas-0.3.21-pthreads_h78a6416_3
---
>  Linking ncurses-6.3-h27087fc_1
1082,1085c1103
<  Linking c-ares-1.18.1-h7f98852_0
<  Linking keyutils-1.6.1-h166bdaf_0
<  Linking openssl-3.1.0-hd590300_2
<  Linking libaec-1.0.6-hcb278e6_1
---
>  Linking icu-70.1-h27087fc_0
1091a1110,1111
>  Linking openblas-0.3.21-pthreads_h320a7e8_3
>  Linking libblas-3.9.0-16_linux64_openblas
1096,1097d1115
<  Linking openblas-0.3.21-pthreads_h320a7e8_3
<  Linking libblas-3.9.0-16_linux64_openblas
1100a1119
>  Linking zstd-1.5.2-h3eb15da_6
1101a1121
>  Linking libnghttp2-1.52.0-h61bc06f_0
1103d1122
<  Linking zstd-1.5.2-h3eb15da_6
1107d1125
<  Linking libnghttp2-1.52.0-h61bc06f_0
1111a1130,1131
>  Linking libcblas-3.9.0-16_linux64_openblas
>  Linking liblapack-3.9.0-16_linux64_openblas
1115,1119d1134
<  Linking libcblas-3.9.0-16_linux64_openblas
<  Linking liblapack-3.9.0-16_linux64_openblas
<  Linking libxslt-1.1.37-h873f0b0_0
<  Linking nss-3.89-he45b914_0
<  Linking sqlite-3.40.0-h4ff8645_1
1122a1138,1140
>  Linking libxslt-1.1.37-h873f0b0_0
>  Linking nss-3.89-he45b914_0
>  Linking sqlite-3.40.0-h4ff8645_1
1131d1148
<  Linking libpq-15.2-hb675445_0
1132a1150
>  Linking libpq-15.2-hb675445_0
1138d1155
<  Linking postgresql-15.2-h3248436_0
1139a1157
>  Linking curl-8.0.1-h588be90_0
1142d1159
<  Linking curl-8.0.1-h588be90_0
1143a1161
>  Linking postgresql-15.2-h3248436_0
1145a1164
>  Linking tiledb-2.13.2-hd532e3d_0
1148,1149d1166
<  Linking tiledb-2.13.2-hd532e3d_0
<  Linking kealib-1.5.0-ha7026e8_0
1150a1168
>  Linking kealib-1.5.0-ha7026e8_0
1166d1183
<  Linking pyshp-2.3.1-pyhd8ed1ab_0
1168,1170d1184
<  Linking munkres-1.1.4-pyh9f0ad1d_0
<  Linking pyparsing-3.0.9-pyhd8ed1ab_0
<  Linking cycler-0.11.0-pyhd8ed1ab_0
1172,1174d1185
<  Linking pytz-2023.3-pyhd8ed1ab_0
<  Linking python-tzdata-2023.3-pyhd8ed1ab_0
<  Linking affine-2.4.0-pyhd8ed1ab_0
1181d1191
<  Linking certifi-2022.12.7-pyhd8ed1ab_0
1182a1193,1200
>  Linking pyshp-2.3.1-pyhd8ed1ab_0
>  Linking munkres-1.1.4-pyh9f0ad1d_0
>  Linking pyparsing-3.0.9-pyhd8ed1ab_0
>  Linking cycler-0.11.0-pyhd8ed1ab_0
>  Linking pytz-2023.3-pyhd8ed1ab_0
>  Linking python-tzdata-2023.3-pyhd8ed1ab_0
>  Linking certifi-2022.12.7-pyhd8ed1ab_0
>  Linking affine-2.4.0-pyhd8ed1ab_0
1226c1244
<  Linking platformdirs-3.3.0-pyhd8ed1ab_0
---
>  Linking platformdirs-3.5.0-pyhd8ed1ab_0
1230c1248
<  Linking virtualenv-20.22.0-pyhd8ed1ab_0
---
>  Linking virtualenv-20.23.0-pyhd8ed1ab_0
1234,1237d1251
<  Linking unicodedata2-15.0.0-py310h5764c6d_0
<  Linking pillow-9.4.0-py310h023d228_1
<  Linking kiwisolver-1.4.4-py310hbf28c38_1
<  Linking llvmlite-0.39.1-py310h58363a5_1
1246d1259
<  Linking libcf-1.0.3-py310h71500c5_116
1247a1261,1264
>  Linking unicodedata2-15.0.0-py310h5764c6d_0
>  Linking pillow-9.4.0-py310h023d228_1
>  Linking kiwisolver-1.4.4-py310hbf28c38_1
>  Linking llvmlite-0.39.1-py310h58363a5_1
1248a1266
>  Linking libcf-1.0.3-py310h71500c5_116
1254c1272
<  Linking coverage-7.2.3-py310h1fa729e_0
---
>  Linking coverage-7.2.4-py310h2372a71_0
1259,1260d1276
<  Linking contourpy-1.0.7-py310hdf3cbec_0
<  Linking shapely-2.0.1-py310h8b84c32_0
1261a1278,1279
>  Linking shapely-2.0.1-py310h8b84c32_0
>  Linking contourpy-1.0.7-py310hdf3cbec_0
1277,1278c1295
<  Linking hypothesis-6.74.0-pyha770c72_0
<  Linking snuggs-1.4.7-py_0
---
>  Linking hypothesis-6.74.1-pyha770c72_0
1279a1297
>  Linking snuggs-1.4.7-py_0
1292c1310
<  Linking requests-2.28.2-pyhd8ed1ab_1
---
>  Linking requests-2.29.0-pyhd8ed1ab_0
1458c1476
<    coverage                   7.2.3         py310h1fa729e_0          conda-forge
---
>    coverage                   7.2.4         py310h2372a71_0          conda-forge
1499c1517
<    hypothesis                 6.74.0        pyha770c72_0             conda-forge
---
>    hypothesis                 6.74.1        pyha770c72_0             conda-forge
1606c1624
<    platformdirs               3.3.0         pyhd8ed1ab_0             conda-forge
---
>    platformdirs               3.5.0         pyhd8ed1ab_0             conda-forge
1640c1658
<    requests                   2.28.2        pyhd8ed1ab_1             conda-forge
---
>    requests                   2.29.0        pyhd8ed1ab_0             conda-forge
1671c1689
<    virtualenv                 20.22.0       pyhd8ed1ab_0             conda-forge
---
>    virtualenv                 20.23.0       pyhd8ed1ab_0             conda-forge
1717c1735
<  [36;1mecho "Contender: 0f4e99d036b0d6d76a3271e6191eacbc9922662f"[0m
---
>  [36;1mecho "Contender: a220022e7ef8d3df68619643954352fa39394ea8"[0m
1723c1741
<  [36;1masv continuous $ASV_OPTIONS v2023.04.2  0f4e99d036b0d6d76a3271e6191eacbc9922662f \[0m
---
>  [36;1masv continuous $ASV_OPTIONS v2023.04.2  a220022e7ef8d3df68619643954352fa39394ea8 \[0m
1744c1762
<  · No information stored about machine 'fv-az133-41'. I know about nothing.
---
>  · No information stored about machine 'fv-az613-176'. I know about nothing.
1754c1772
<  machine [fv-az133-41]: 2. os:  The OS type and version of this machine.  For example,
---
>  machine [fv-az613-176]: 2. os:  The OS type and version of this machine.  For example,
1756c1774
<  os [Linux 5.15.0-1035-azure]: 3. arch:  The generic CPU architecture of this machine.  For example,
---
>  os [Linux 5.15.0-1036-azure]: 3. arch:  The generic CPU architecture of this machine.  For example,
1761c1779
<  cpu [Intel(R) Xeon(R) Platinum 8370C CPU @ 2.80GHz]: 5. num_cpu:  The number of CPUs in the system. For example, '4'.
---
>  cpu [Intel(R) Xeon(R) Platinum 8171M CPU @ 2.60GHz]: 5. num_cpu:  The number of CPUs in the system. For example, '4'.
1765,1767c1783,1785
<  ram [7110636]: Baseline: v2023.04.2 
<  + echo 'Contender: 0f4e99d036b0d6d76a3271e6191eacbc9922662f'
<  Contender: 0f4e99d036b0d6d76a3271e6191eacbc9922662f
---
>  + echo 'Contender: a220022e7ef8d3df68619643954352fa39394ea8'
>  ram [7110632]: Baseline: v2023.04.2 
>  Contender: a220022e7ef8d3df68619643954352fa39394ea8
1772d1789
<  + asv continuous --split --show-stderr --factor 1.5 v2023.04.2 0f4e99d036b0d6d76a3271e6191eacbc9922662f
1773a1791
>  + asv continuous --split --show-stderr --factor 1.5 v2023.04.2 a220022e7ef8d3df68619643954352fa39394ea8
1774a1793,1818
>  Traceback (most recent call last):
>    File "/home/runner/micromamba-root/envs/xarray-tests/bin/asv", line 10, in <module>
>      sys.exit(main())
>    File "/home/runner/micromamba-root/envs/xarray-tests/lib/python3.10/site-packages/asv/main.py", line 38, in main
>      result = args.func(args)
>    File "/home/runner/micromamba-root/envs/xarray-tests/lib/python3.10/site-packages/asv/commands/__init__.py", line 49, in run_from_args
>      return cls.run_from_conf_args(conf, args)
>    File "/home/runner/micromamba-root/envs/xarray-tests/lib/python3.10/site-packages/asv/commands/continuous.py", line 75, in run_from_conf_args
>      return cls.run(
>    File "/home/runner/micromamba-root/envs/xarray-tests/lib/python3.10/site-packages/asv/commands/continuous.py", line 114, in run
>      result = Run.run(
>    File "/home/runner/micromamba-root/envs/xarray-tests/lib/python3.10/site-packages/asv/commands/run.py", line 294, in run
>      Setup.perform_setup(environments, parallel=parallel)
>    File "/home/runner/micromamba-root/envs/xarray-tests/lib/python3.10/site-packages/asv/commands/setup.py", line 89, in perform_setup
>      list(map(_create, environments))
>    File "/home/runner/micromamba-root/envs/xarray-tests/lib/python3.10/site-packages/asv/commands/setup.py", line 21, in _create
>      env.create()
>    File "/home/runner/micromamba-root/envs/xarray-tests/lib/python3.10/site-packages/asv/environment.py", line 704, in create
>      self._setup()
>    File "/home/runner/micromamba-root/envs/xarray-tests/lib/python3.10/site-packages/asv/plugins/conda.py", line 174, in _setup
>      self._run_conda(['env', 'create', '-f', env_file_name,
>    File "/home/runner/micromamba-root/envs/xarray-tests/lib/python3.10/site-packages/asv/plugins/conda.py", line 227, in _run_conda
>      return util.check_output([conda] + args, env=env)
>    File "/home/runner/micromamba-root/envs/xarray-tests/lib/python3.10/site-packages/asv/util.py", line 754, in check_output
>      raise ProcessError(args, retcode, stdout, stderr)
>  asv.util.ProcessError: Command '/usr/bin/conda env create -f /tmp/tmp4i32rw09.yml -p /home/runner/work/xarray/xarray/asv_bench/.asv/env/df282ba4a530a0853b7f9108ec3ff02d --force' returned non-zero exit status 1
1776,1827c1820,1826
<  · Discovering benchmarks
<  ·· Uninstalling from conda-py3.10-bottleneck-cftime-dask-distributed-flox-netcdf4-numpy-numpy_groupies-pandas-scipy-setuptools_scm-sparse
<  ·· Building 0f4e99d0 <main> for conda-py3.10-bottleneck-cftime-dask-distributed-flox-netcdf4-numpy-numpy_groupies-pandas-scipy-setuptools_scm-sparse
<  ·· Installing 0f4e99d0 <main> into conda-py3.10-bottleneck-cftime-dask-distributed-flox-netcdf4-numpy-numpy_groupies-pandas-scipy-setuptools_scm-sparse
<  · Running 362 total benchmarks (2 commits * 1 environments * 181 benchmarks)
<  [  0.00%] · For xarray commit 91f14c9b <v2023.04.2> (round 1/2):
<  [  0.00%] ·· Building for conda-py3.10-bottleneck-cftime-dask-distributed-flox-netcdf4-numpy-numpy_groupies-pandas-scipy-setuptools_scm-sparse

```

**Comment by dcherian** [comment]
`asv run -v` failed locally and printed out the yaml file. For some reason `channels` is empty
```
·· Error running /Users/dcherian/mambaforge/bin/conda env create -f /var/folders/yp/rzgd0f7n5zbcbf5dg73sfb8ntv3xxm/T/tmpbusmg2a0.yml -p /Users/dcherian/work/python/xarray/asv_bench/.asv/env/df282ba4a530a0853b7f9108ec3ff02d --force (exit status 1)
·· conda env create/update failed: in /Users/dcherian/work/python/xarray/asv_bench/.asv/env/df282ba4a530a0853b7f9108ec3ff02d with:
   name: conda-py3.10-bottleneck-cftime-dask-distributed-flox-netcdf4-numpy-numpy_groupies-pandas-scipy-setuptools_scm-sparse
   channels:
   dependencies:
      - python=3.10
      - wheel
      - pip
      - setuptools_scm
      - numpy
      - pandas
      - netcdf4
      - scipy
      - bottleneck
      - dask
      - distributed
      - flox
      - numpy_groupies
      - sparse
      - cftime
  ```

File: .github/workflows/benchmarks.yml [hunk 2]
```diff
@@ -45,6 +45,7 @@ jobs:
           set -x
           # ID this runner
           asv machine --yes
+          export TMPDIR=$RUNNER_TEMP
```
Comment (dcherian) [inline .github/workflows/benchmarks.yml:48]:
```suggestion
```

**Comment by dcherian** [comment]
well that was it apparently 🤷🏾‍♂️

File: asv_bench/benchmarks/groupby.py [hunk 3]
```diff
@@ -132,3 +141,34 @@ def setup(self, *args, **kwargs):
...
```
Comment (Illviljan) [inline asv_bench/benchmarks/groupby.py:146]:
```suggestion
class ResampleCFTime(Resample):
```
This class has currently no tests, is this what was intended?

File: asv_bench/benchmarks/groupby.py [hunk 4]
```diff
@@ -132,3 +141,34 @@ def setup(self, *args, **kwargs):
...
+            },
+        )
+        self.ds2d = self.ds1d.expand_dims(z=10)
+        self.ds1d_mean = self.ds1d.resample(time="48H").mean()
+        self.ds2d_mean = self.ds2d.resample(time="48H").mean()
+
+
+class GroupByLongTime:
+    params = [[True, False], [True, False]]
+    param_names = ["use_cftime", "use_flox"]
```
Comment (Illviljan) [inline asv_bench/benchmarks/groupby.py:165]: We usually use the parameterized decorator?

**Comment by dcherian** [comment]
Thanks @Illviljan I'm merging so I can run benchmarks on a few other PRs

**Comment by trexfeathers** [comment]
If you're curious what happened, we had the same problem: https://github.com/SciTools/iris/issues/5280#issuecomment-1525802077

Just wish I'd spotted this sooner but it's quite hard to follow two organisations' repos 😆
